src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nr21-03.mdx

```diff
@@ -56,13 +56,13 @@ The following New Relic Java agent versions are affected:
 
 To remediate this vulnerability in the New Relic Java agent, we recommend customers take one of the following actions immediately:
 
-1. Upgrade your Java Agent to either version [6.5.1](/docs/release-notes/agent-release-notes/java-release-notes/java-agent-651/) or [7.4.1](/docs/release-notes/agent-release-notes/java-release-notes/java-agent-741/). These agent versions have been updated to use the remediated 2.15.0 version of the Apache Log4j framework.
+1. Upgrade your Java Agent to either version [6.5.2](/docs/release-notes/agent-release-notes/java-release-notes/java-agent-651/) or [7.4.2](/docs/release-notes/agent-release-notes/java-release-notes/java-agent-741/). These agent versions have been updated to use the remediated 2.15.0 version of the Apache Log4j framework.
```

Comment:
I see that you have changed the version numbers but the links still point to 651 and 741. Is this intentional? Thanks for confirming!